lua/textcase/plugin/presets.lua

```diff
@@ -46,6 +46,9 @@ local function setup_default_keymappings()
     { method_name = "to_pascal_case", quick_replace = "p", operator = "op", lsp_rename = "P" },
     { method_name = "to_upper_case", quick_replace = "u", operator = "ou", lsp_rename = "U" },
     { method_name = "to_lower_case", quick_replace = "l", operator = "ol", lsp_rename = "L" },
+    { method_name = "to_title_case", quick_replace = "t", operator = "ot", lsp_rename = "T" },
+    { method_name = "to_dot_case", quick_replace = "o", operator = "oo", lsp_rename = "O" },
```

Comment:
Could you comment these two lines of codes (mappings for to_dot_case and to_path_case) and add a comment on why they are not added as defaults?

I think such comment will be a guide for future contributors interested on the reason why they aren't added by default

Reply:
Sure! It's done.